.azure-pipelines/client.yml

```diff
@@ -1,3 +1,8 @@
+# External variables:
+# packageSuffix
+#   The package suffix for the package to build and test. Defined in pipeline web ui because multiple pipelines use this YAML with different packages.
+#   Example: -eventhubs-client
```

Comment:
It is kind of unfortunate that we need to include the "-" I wonder if we can do this by pre-pending this to if the value isn't empty?